potlako_subject/models/signals.py

```diff
@@ -127,7 +127,10 @@ def patient_call_followup_on_post_save(sender, instance, raw, created, **kwargs)
                 except SubjectConsent.DoesNotExist:
                     raise ValidationError('Subject screening object does not exist!')
                 else:
-                    if (instance.next_appointment_date and get_community_arm(
+                    appt_status = instance.subject_visit.appointment.next_by_timepoint.appt_status
```

Comment:
this will throw an attributeerror for the last appointment (3000), since next_by_timepoint will return a NoneType for 3000.